development/build/utils.js

```diff
@@ -1,8 +1,18 @@
 const path = require('path');
+const { readFileSync } = require('fs');
 const semver = require('semver');
+const { capitalize } = require('lodash');
 const { loadBuildTypesConfig } = require('../lib/build-type');
 const { BUILD_TARGETS, ENVIRONMENT } = require('./constants');
 
+const BUILD_TYPES_TO_SVG_LOGO_PATH = {
+  main: './app/images/logo/metamask-fox.svg',
+  beta: './app/build-types/beta/images/logo/metamask-fox.svg',
+  flask: './app/build-types/flask/images/logo/metamask-fox.svg',
+  mmi: './app/build-types/mmi/images/logo/mmi-logo.svg',
+  desktop: './app/build-types/desktop/images/logo/metamask-fox.svg',
```

Comment:
didn't know about the desktop build? we still making that?

Reply:
dunno. it's one of the possible build types and we have assets for it, so I included it just in case